layer/nrlf/nrlf/core/common_producer_steps.py

```diff
@@ -37,3 +44,33 @@ def validate_producer_permissions(
 def invalid_producer_for_delete(ods_code_parts: tuple[str], delete_item_id: str):
     producer_id, _ = generate_producer_id(id=delete_item_id, producer_id=None)
     return tuple(producer_id.split(CUSTODIAN_SEPARATOR)) != ods_code_parts
+
+
+@log_action(log_reference=LogReference.COMMONPRODUCER002)
+def apply_json_schema_validators(
+    data: PipelineData,
+    context: LambdaContext,
+    event: APIGatewayProxyEventModel,
+    dependencies: FrozenDict[str, Any],
+    logger: Logger,
+) -> PipelineData:
+    core_model: DocumentPointer = data["core_model"]
+    system, value = split_pointer_type(core_model.type.__root__)
+    json_schema_validators: JsonSchemaMapping = dependencies["json_schema_validators"]
+    repository: Repository = dependencies["contract_repository"]
+
+    default_validators = json_schema_validators.get_default() or get_validators_from_db(
```

Comment:
Is this the caching method? Im a little confused by what the default part means

Reply:
Introduced better naming so that it's clear that this is what is happening